src/EdFi.AnalyticsMiddleTier.Tests/.env.example

```diff
@@ -1,12 +1,13 @@
 USE_MSSQL_DEFAULT_CONN_STRING=TRUE
 USE_POSTGRES_DEFAULT_CONN_STRING=TRUE
-SQLSERVER_SERVER=localhost
```

Comment:
Similar changes will affect `.env.example.docker` file. I think you have to rebase your branch to see that file.